neuro_extras/main.py

```diff
@@ -452,13 +403,20 @@ async def _nonstorage_cp(source_url: URL, destination_url: URL) -> None:
         if source_path.is_dir():
             dir_util.remove_tree(str(source_path))
         else:
-            source_path.unlink()
+            if source_path.exists():
+                source_path.unlink()
+
+
+@main.group()
```

Comment:
Could you please move this command group definition above its implementation?

Reply:
It can only go below the implementation, but you're right — it's more clear to have them next to each other